docs/code/mecsimcalc-library.md

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_label: "mecsimcalc Library"
+sidebar_position: 4
+---
+
+# Mecsimcalc 0.0.2 documentation
+
+This library is designed to provide a set of functions for handling and converting various types of data, such as base64 encoded data, Pandas DataFrames, and Pillow images.
+
+```python
+from PIL import Image
+import base64
+import io
+import pandas as pd
+```
+
+## General
+
+### decode_file_data
+
+```python
+def decode_file_data(encoded_data, metadata)
+```
+
+#### Description:
+
+Converts a base64 encoded file data into a file object. (e.g. converts data from **`file = inputs['fileInput']`** to a file object)
+
+#### Arguments:
+
+- **`encoded_data`** (str): Base64 encoded file data
+- **`metadata`** (bool, optional): If True, returns a tuple of (fileData, metadata). Defaults to False
+
+#### Returns:
+
+- **`io.BytesIO`**: fileData
+- **`(io.BytesIO, str)`**: (fileData, metadata) if metadata is True
+
```

Comment:
For all the functions, can you add an example of how it's used, and have the example value of the input/output variables as comments.

Reply:
Kinda like how numpy docs have the examples:
https://numpy.org/doc/stable/reference/generated/numpy.asarray.html